recipes/repository.rb

```diff
@@ -19,7 +19,22 @@
 
 case node['platform_family']
 when 'fedora', 'rhel'
```

Comment:
amazon linux here is `rhel` ? asking cuz I don't know wether it has a separate platform_family or not.